src/runtime/metatype.cs

```diff
@@ -360,4 +356,18 @@ public static IntPtr __subclasscheck__(IntPtr tp, IntPtr args)
             return DoInstanceCheck(tp, args, true);
         }
     }
+
+
+    [StructLayout(LayoutKind.Sequential)]
+    struct ClrMetaTypeEx
+    {
+        public IntPtr ClrHandleOffset;
+        public IntPtr ClrHandle;
```

Comment:
These two refer to different handles, yet are named similarly, which is confusing.

Besides, in C# 9 we can start using `nint` instead of `IntPtr` for non-pointers.

Reply:
What name do you prefer?

Reply:
This is just a NIT. Maybe `TypeClrHandle` and `InstanceClrHandleOffset`?